src/main/java/org/apache/maven/plugins/pmd/PmdReport.java

```diff
@@ -577,13 +579,19 @@ private String determineAuxClasspath() throws MavenReportException
                         dependencies.add( resolvedArtifact.getArtifact().getFile().toString() );
                     }
 
-                    List<String> projectCompileClasspath = includeTests ? localProject.getTestClasspathElements()
-                            : localProject.getCompileClasspathElements();
+                    List<String> projectClasspath = new ArrayList<>(
```

Comment:
I think, this change is not needed: `getTestClasspathElements()` contains already `getCompileClasspathElements()` - so we would have e.g. `target/classes` now twice on the classpath